### ISSUE TITLE
Network Extension: WeTransfer uploads stuck at 100%

### ISSUE THREAD

**Issue by iDjay** [AdguardTeam/AdguardForMac#760]
Since switching to the new network extension mode in AdGuard 2.5, my weTransfer uploads get stuck at 100%. Anyone else having this issue?

Catalina 10.15.7

Steps to reproduce:

- open weTransfer.com
- try to upload a file which is a bit bigger
- see if it's uploading successful or getting stuck at 100%

**Comment by fhavrlent** [comment]
I might have similar issues, but with MS Teams (can call and send messages, but won't receive) and when pulling from docker registry, it fails. Using Catalina, 10.15.7.

**Comment by Chinaski1** [comment]
Hello there!

I tried to reproduce your issue but got [no result](https://uploads.adguard.com/Chinaskij29yq.png).
Could you clarify whether your MAC uses side software? (VPN/Antivirus)
- I can also recommend you to check for Nightly versions of the product (agrd.io/mac_nightly).

**Comment by iDjay** [comment]
Forgot to mention, I'm using Safari, didn't try a different browser yet.

**Comment by fhavrlent** [comment]
In my case it was not question of browser, I was not able to pull docker images from registry via command line, the connection just erred. I will try it again, maybe it was updated, but I had to switch back to kernel extension. 

I'm using Tunnelbrick as VPN client, but I don't think there was any difference if I was connected or disconnected and Mallwarebytes, but just for on-demand scanning (free version).
